### PR TITLE
VideoPress: use American English in Plan description.

### PR DESCRIPTION
**Pull request by jeherve**

### CODE REVIEW

File: help.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+https://github.com/Automattic/wp-calypso/tree/master/docs/guide
```
Comment (dereksmart) [inline help.md]: This seemed to have been added on accident?

Comment (jeherve) [inline help.md]: uhoh. That's not supposed to be there, that's work in progress for better Jetpack docs! I removed it.

**Comment by eliorivero** [comment]
LGTM! 🐑

**Comment by dereksmart** [comment]
in branch-4.5 19f04e7

**Pull request merged**